acceptance/setup/aio/pre-suite/021_InstallAristaModule.rb

```diff
@@ -0,0 +1,18 @@
+platforms = hosts.map{|val| val[:platform]}
+skip_test "No arista hosts present" unless platforms.any? { |val| /eos/ =~ val }
```

Comment:
I think we prefer using `confine :to, :platform => /eos/`, though I defer to @johnduarte and @er0ck on that